### DISCUSSION TITLE
Apple M1 AG VST and Au not present in Ableton 11

### DISCUSSION THREAD

**Discussion by nicoc83** [apohl79/audiogridder#437] [Help] [answered]
hi there,

today i installed the audiogridder for my Mac with M1 
I installed the arm version 1.1.0 and 1.1.1 but both of them i didnt see in ableton.
The vst and au files are in the right place.
when i install the intel version i only see au and vst3 but both with over 200 percent processor activity when loading
anyone have an idea ? :)

thanks for the great software 
nico

**Comment by apohl79** [comment, marked as answer]
You can only load the arm64 version of the plugin into a host supporting that same architecture. Ableton does not support arm yet, so you need rosetta and that also means you can only load the x86 versions of AG.

**Reply by nicoc83** [reply]
ah ok, thx 
then i will try the intel version and hope i can reduce the processor activity

**Reply by Teebone2000** [reply]
I have the same problem, but I cannot get the x86 version of AG to show up in Ableton on my m1 at all?
